io500-info-editor.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+import sys
+import json
+
+# This tool allows to view and edit the options in an existing schema file.
+# In order to get started download the two files:
+# wget https://www.vi4io.org/lib/plugins/newcdcl/scripts/schema-io500.json
```

Comment:
I actually have a bunch of comments on the schema, is there anywhere I could post comments?

Reply:
In the Google Doc that I shared a while ago for the IO500 committee.